xarray/core/duck_array_ops.py

```diff
@@ -129,7 +136,7 @@ def notnull(data):
 
 
 def gradient(x, coord, axis, edge_order):
-    if isinstance(x, dask_array_type):
+    if is_duck_dask_array(x):
         return dask_array.gradient(x, coord, axis=axis, edge_order=edge_order)
```

Comment:
is this still necessary, or does `dask` define `numpy.gradient` via `__array_function__`?

Reply:
Our minimum numpy version does not support NEP-18 so this (and all of duck_array_ops) is still necessary?

Reply:
ah, right, I forgot about that.